app/components/markdown-block.js

```diff
@@ -1,43 +1,221 @@
 //https://github.com/EnCiv/enciv-home/issues/41
+//https://github.com/EnCiv/enciv-home/issues/markDownBlock-withImage#45
```

Comment:
This url doesn't work.  It should be in the same format as the line above.  You should be able to pasted it into a browser, or click on it in VSC and get to the issue.